prow/cmd/deck/main.go

```diff
@@ -1338,6 +1333,7 @@ func handleRerun(prowJobClient prowv1.ProwJobInterface, createProwJob bool, cfg
 					l.WithError(err).Errorf("Error checking if user can trigger job")
 					return
 				}
+				logrus.Info(fmt.Sprintf("Attempted rerun: user: %s, job: %s, allowed: %t", login, newPJ.Spec.Job, allowed))
```

Comment:
Please use log fields so that we can filter on user, job, and allowed. (`logrus.WithFields(...).Info(...)`)